vagrant/post-deploy.d/00-run-docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eu
+
+PAAS_DIR=/tmp/paas-bootstrap
+
+sudo apt update && sudo apt install docker-compose -y
+
+git clone https://github.com/alphagov/paas-bootstrap.git ${PAAS_DIR}
```

Comment:
The vagrant directory is available on the vagrant box. If you copy the required files here, no need to use git.